### DISCUSSION TITLE
Hosting con dominio gratuito

### DISCUSSION THREAD

**Discussion by doguedogue** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#171] [Ideas]
Para subir tu página hay muchos lugares gratuitos.

- GitHub pages
- W3schools https://www.w3schools.com/spaces/
- Heroku
- Google pages
- Etc.

Las palabras claves: hosting, free, subir páginas, etc.

Un ejemplo de como queda con w3schools: https://doguedogue.w3spaces.com/

**Comment by hackit92** [comment]
Complementando lo que comentas, recomendaría utilizar AWS u Oracle Free Tier. Igual hay alojamientos de muy bajo costo que pueden ser excelentes para comenzar a practicar en un entorno real
